core/module.h

```diff
@@ -327,6 +331,10 @@ class alignas(64) Module {
 
   const std::vector<bess::OGate *> &ogates() const { return ogates_; }
 
+  uint64_t deadends() const {
```

Comment:
I guess we want per-worker statistics here either array or map. The caller (`bessctl.cc`) able to accumulation if required.

Reply:
In my head `deadends_` is just an array to make the counter thread-safe-- i.e., it's just an implementation detail. I only ever intended to expose a single counter to the user here